main.pm

```diff
@@ -196,6 +196,11 @@ sub gnomestep_is_applicable() {
     return check_var("DESKTOP", "gnome");
 }
 
+sub snapper_is_applicable() {
+    my $fs = get_var("FILESYSTEM", 'btrfs');
+    return ( $fs eq "btrfs" && get_var("HDDSIZEGB") > 10);
```

Comment:
@aaannz explained it perfectly. And yes, we need more than 10GB (about 14GB) so snapshots are enabled.